crates/viewer/re_view_spatial/src/view_2d.rs

```diff
@@ -275,32 +280,39 @@ impl ViewClass for SpatialView2D {
     }
 }
 
-#[derive(Default, Debug)]
+#[derive(Default, Copy, Clone, Debug, PartialEq, Eq)]
 struct NonNestedImageCounts {
-    image: usize,
-    encoded_image: usize,
+    color: usize,
     depth: usize,
-    video: usize,
     segmentation: usize,
 }
 
 impl NonNestedImageCounts {
-    fn has_any_images(&self) -> bool {
+    fn total(&self) -> usize {
         let Self {
-            image,
-            encoded_image,
+            color,
             depth,
-            video,
             segmentation,
         } = self;
+        color + depth + segmentation
+    }
 
-        *image > 0 || *encoded_image > 0 || *depth > 0 || *video > 0 || *segmentation > 0
+    fn has_any_images(&self) -> bool {
+        self.total() > 0
+    }
+
+    fn count(&mut self, dims: &MaxDimensions) {
```

Comment:
`count` sounds like it will return something
```suggestion
    fn update_counts(&mut self, dims: &MaxDimensions) {
```